### ISSUE TITLE
bash completion error

### ISSUE THREAD

**Issue by sispus** [cyberang3l/kdeconnect-sms#1]
Hi,
I have problem in bash completion.

I created a .addressBook in home, and did:
sed -i -n -e '/kdeconnect-sms/!p' -e '$a source <(kdeconnect-sms --bash-completion)' ~/.bashrc

then, ı followed the demo video to complete my installation. Now, whenever I hit the tab button, it gives  the error: 
bash: _expand: command not found

**Comment by cyberang3l** [comment]
Did you logout/login to load the contents of the `.bashrc` file?

What's output of the following commands?
```
grep kdeconnect ~/.bashrc
```

```
which kdeconnect-sms
```

**Comment by cyberang3l** [comment]
Maybe you are missing the `_expand` function?

https://askubuntu.com/questions/679575/what-does-expand-do

Does the autocompletion work fine for other executables?
What is the distro that you are testing the script at? I have only tested it on Ubuntu.

**Comment by sispus** [comment]
I tried it on manjaro xfce

```
$ grep kdeconnect ~/.bashrc
source <(kdeconnect-sms --bash-completion)
$ which kdeconnect-sms
/usr/local/bin/kdeconnect-sms
```

I didn't go your second message but I found a program in software management (pamac) named "bash completion" and installed it (with no reason :-)). And made in terminal 
`$ source <(kdeconnect-sms -b)
`
Nothing changed. 

Then I made in terminal: 
`$ source <(/usr/local/bin/kdeconnect-sms -b)
`
and now it works.

Thank you!
